components/sidebar/sidebar.jsx

```diff
@@ -583,6 +589,24 @@ class Sidebar extends React.PureComponent {
                     id='sidebarChannelContainer'
                     className='nav-pills__container'
                 >
+                    {pluginApps && pluginApps.length > 0 &&
+                        <ul
+                            key='apps'
+                            className='nav nav-pills nav-stacked'
+                        >
+                            <li>
+                                <h4 id='apps'>
+                                    <FormattedMessage
+                                        id={'applications'}
```

Comment:
We normally use "scoped" translation keys, I would expect to see something like, `sidebar.applications` instead of just `applications`

Reply:
done

Reply:
Yo have change the `defaultMessage`, which is the string used when no translation is found, I was talking about the `id`